pkg/sql/catalog/bootstrap/kv_writer.go

```diff
@@ -86,25 +89,60 @@ func (w KVWriter) RecordToKeyValues(values ...tree.Datum) (ret []roachpb.KeyValu
 
 // Insert updates a batch with the KV operations required to insert a new record
 // into the table.
-func (w KVWriter) Insert(b *kv.Batch, values ...tree.Datum) error {
+func (w KVWriter) Insert(
```

Comment:
I wonder if we could have this `kvTrace` setting on the struct itself. I don't think it's worth doing now, but it might clean some things up a bit and reduce some surface area.